test/21.js

```diff
@@ -0,0 +1,13 @@
+const expect = require('chai').expect;
+let solution = require('../solutions/21').solution; 
+
+describe('Three largest numbers',() =>{
+  it('should return 9,12,19 for [8,1,4,2,12,6,7,19,2,9]', () => {
+    const result = solution([8,1,4,2,12,6,7,19,2,9]);
+    expect(result).ti.equal(8,12,19);
+  });
```

Comment:
@rkalra247 `` expect(result).to.equal ``
instead of `` expect(result).ti.equal ``